.circleci/config.yml

```diff
@@ -62,10 +62,30 @@ jobs: # a collection of steps
           # Upload test summary for display in Artifacts: https://circleci.com/docs/2.0/artifacts/ 
           path: backend/reports/coverage/lcov-report
           destination: backend-coverage
+      - run:
+          name: install bas-platform
+          working_directory: bas-platform
+          command: npm install
+      - run:
+          name: build bas-platform
+          working_directory: bas-platform
+          command: npm run compile
+      - run:
+          name: create .vsix
+          working_directory: bas-platform
+          command: npx vsce package .
+      # - run:
+      #     name: test bas-platform
+      #     working_directory: bas-platform
+      #     command: npm run test
```

Comment:
Is this commented on purpose because we don't have tests yet?

Reply:
Yes, I'll add an issue to implement tests for bas_platform